src/lib.rs

```diff
@@ -21,43 +20,48 @@ pub fn greet() -> String {
 }
 
 #[wasm_bindgen]
-pub fn draw() {
-    let document = web_sys::window().unwrap().document().unwrap();
-    let canvas = document.get_element_by_id("canvas").unwrap();
-    let canvas: web_sys::HtmlCanvasElement = canvas
-        .dyn_into::<web_sys::HtmlCanvasElement>()
-        .map_err(|_| ())
-        .unwrap();
+pub fn draw(up: bool, _down: bool, _left: bool, _right: bool) -> Option<bool> {
```

Comment:
Params = why?

Reply:
I get it now but also complete your thoughts!

Reply:
Added comment